### ISSUE TITLE
Support ESModule

### ISSUE THREAD

**Issue by JeanMeche** [brightlayer-ui/colors#47]
Currently, when using `@brightlayer-ui/colors` with angular, the compiler throws the following warning : 

```
XXXX.js depends on '@brightlayer-ui/colors'. CommonJS or AMD dependencies can cause optimization bailouts.
```

If i'm correct, this can be fixed by shipping a ES Module version of the package. 

Could you look into it ? 

Thx

**Comment by emclaug2** [comment]
This now fixed in [@brightlayer-ui/colors@3.1.0](https://www.npmjs.com/package/@brightlayer-ui/colors/v/3.1.0); marking this issue as closed now.

**Comment by emclaug2** [comment]
Hey @JeanMeche, we unfortunately had to revert this change since it had some unintended effects on other frameworks' testing environments.  This new build module didn't integrate nicely into our React testing library's Jest testing environment.  The colors package has been reverted to, by default, use a commonjs module.  We did, however, keep the es2015 module as part of the package which can be consumed by your angular app.

If you update your `tsconfig.json` to use a custom path resolution, you could continue to use the es2015 module instead of the default commonjs one.

Here is an example code snippet: 
```

  "compilerOptions": {
    "paths": {
      "@brightlayer-ui/colors": ["../node_modules/@brightlayer-ui/colors/dist/es2015"]
    }
  },
```

This patch is available in @brightlayer-ui/colors@3.1.1.

**Comment by JeanMeche** [comment]
That does the job, thanks !
